README.md

```diff
@@ -912,6 +912,31 @@ params do
 end
 ```
 
+If type is Array, It support following kinds of parameter.
```

Comment:
We never use `curl` in examples here, I don't think it's a good idea to introduce it. It's also fairly confusing TBH given that you POST to a JSON API, but don't actually send JSON data. I would explain this in Ruby, keeping it more like "documentation" and less like "example"?